### ISSUE TITLE
Forcefully setting aws_ami

### ISSUE THREAD

**Issue by mheppner** [dcos-terraform/terraform-aws-tested-oses#12]
I'm trying to get the universal installer to work on GovCloud (a different partition than standard "aws"). After making some local modifications to other modules using sed, I cannot seem to get this module to follow the value set for `aws_ami`.

My main.tf:
```terraform
module "dcos" {
  source = "dcos-terraform/dcos/aws"
  version = "~> 0.2.3"
  providers = {
    aws = "aws"
  }
  ...

  aws_ami = "ami-<id>" # custom AMI deployed in GovCloud
  dcos_instance_os = "centos_7.5"
}
```

What I'm trying to do is to replace [this line](https://github.com/dcos-terraform/terraform-aws-tested-oses/blob/0.2.0/main.tf#L29) with sed:
```terraform
aws_ami_result = "${lookup(var.aws_ami, format("%s_%s",var.os, coalesce(var.region, data.aws_region.current.name)))}"
# replace the above line to this line:
aws_ami_result = "${var.aws_ami}"
```

Basically, just force the module to use the value set in `aws_ami` directly. However, I realized that the `lookup()` should be doing this already, using the provided value and defaulting to the constructed one above, correct?

If I replace the line above, I get an error that `aws_ami` is a map and needs to be a string, which makes me think that's why the `lookup()` above fails.

**Comment by Promaethius** [comment]
Looking at the community AMI list, I got `centos_7.6` as a working entry.
